struqture-py/src/mixed_systems/mixed_product.rs

```diff
@@ -212,4 +212,61 @@ impl MixedProductWrapper {
         }
         Ok(return_vector)
     }
+
+    pub fn pprint(&self) -> String {
+        let mut output = String::new();
+        // Spins
+        match self.spins().len() {
+            0 => {}
+            1 => output.push_str((self.spins()[0].pprint() + " ").as_str()),
+            _ => {
+                output.push('[');
```

Comment:
Using brakets here (and further down) to group the different spin, fermion and boson subsystems could lead to a missunderstanding in the special case that there are two subsystems, that we are trying to print out the commutator of the operators. I would suggest replacing brakets with parentheses here.

Reply:
Good point, thank you!